### PR TITLE
On update race condition

### PR DESCRIPTION
**Pull request by brocollie08**
### Change Type (required)
Indicate the type of change your pull request is:

- [ ] `patch`
- [ ] `minor`
- [ ] `major`

### Does your PR have any documentation updates?
- [ ] Updated docs
- [ ] No Update needed
- [ ] Unable to update docs

<details>
  <summary>📦 Published PR as canary version: <code>0.8.1--canary.471.16682</code></summary>
  <br />

  Try this version out locally by upgrading relevant packages to 0.8.1--canary.471.16682
</details>

### CODE REVIEW

File: core/player/src/view/view.ts [hunk 1]
```diff
@@ -94,21 +96,29 @@ export class ViewInstance implements ValidationProvider {
...
     this.resolverOptions = resolverOptions;
     this.hooks.onTemplatePluginCreated.tap("view", (templatePlugin) => {
       this.templatePlugin = templatePlugin;
     });
+    resolverOptions.flowController.current?.hooks.transition.tap("view", (state1, state2) => {
+      this.transitioning = true;
+    })
+    resolverOptions.flowController.current?.hooks.afterTransition.tap("view", (flowInstance) => {
+      this.transitioning = false;
+    })
```
Comment (intuit-svc) [inline core/player/src/view/view.ts:110]: You'll want to tap the `flow` hook on the `flowController` instead of using `current`

Comment (adierkens) [inline core/player/src/view/view.ts:110]: You'll want to use the `.flow` hook on the flow-controller instead of using `current` directly.

Comment (brocollie08) [reply core/player/src/view/view.ts]: while accessible. the call of that hook takes place earlier than the tap due to flow/view scope diff, so I moved the logic out to the view controller

**Review by sugarmanz — APPROVED** [review]

**Pull request merged**